pkg/updater/gcs.go

```diff
@@ -390,12 +391,17 @@ func convertResult(log logrus.FieldLogger, nameCfg nameConfig, id string, header
 
 	emailAddressesInterface, ok := result.finished.Finished.Metadata[EmailListKey]
 	if ok {
-		emailAddresses, ok := emailAddressesInterface.([]string)
```

Comment:
Do we have unit tests that cover this scenario? Can we add them?

Reply:
Not sure I understand, It's what I removed